src/adagrams.js

```diff
@@ -0,0 +1,4 @@
+const Adagrams = {
+};
+
+export default Adagrams;
```

Comment:
We never explicitly write down (though we usually say it verbally) to leave the line at the bottom of a file. Maybe this is a good opportunity for us to leave a comment like "hey leave this at the bottom of the file and don't delete it!"